packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -325,6 +324,7 @@ export default function DataviewsPatterns() {
 				elements: SYNC_FILTERS,
 				filterBy: {
 					operators: [ OPERATOR_IN ],
+					isPrimary: true,
```

Comment:
Should this be inside `filterBy`? I think it's more of top level filters related prop.

Reply:
Same thought.

Reply:
I did consider having it top-level, and it looked like this:

```js
{
  header: 'Sync',
  id: 'sync',
  isPrimaryFilter: true,
}
```

It's weird that a field says "I'm a primary filter".

Additionally, fields only have `filterBy` prop related to filters. It's also the reason `filterBy` is an object, so we can add more props as we evolve the filters API. I prefer to group it together, to be honest. If the `filterBy` name is confusing, I'd be open to find alternatives such as `filters`, etc. in follow-ups.

Reply:
Let's probably rename then in a follow up(? 🤔 ).